docs/en/02_Developer_Guides/01_Templates/05_Template_Inheritance.md

```diff
@@ -139,3 +80,97 @@ If your classes have in a namespace, the Layout folder will be a found inside of
 
 For example, the layout template for `SilverStripe\Control\Controller` will be
 found at `templates/SilverStripe/Control/Layout/Controller.ss`.
+
+## Cascading themes
+
+Within each theme or templates folder, a specific path representing a template can potentially be found. As
+there may be multiple instances of any matching path for a template across the set of all themes, a cascading
+search is done in order to determine the resolved template for any specified string.
+
+In order to declare the priority for this search, themes can be declared in a cascading fashion in order
+to determine resolution priority. This search is based on the following three configuration values:
+
+ - `SilverStripe\View\SSViewer.themes` - The list of all themes in order of priority. This includes the default
+   set via `$default` as a theme set. This config is normally set by the web developer.
+ - `SilverStripe\Core\Manifest\ModuleManifest.module_priority` - The priority of modules within which $default
+   theme templates should be sorted. This config is normally set by the module author, and does not normally need
```

Comment:
is this a list/map/dictionary of priorities?

Reply:
List.